tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "experimentalDecorators": true,
+    "skipLibCheck": true,
```

Comment:
I don't know how we can avoid this error without `skipLibCheck`:

https://github.com/carbon-design-system/carbon-web-components/runs/4325309889?check_suite_focus=true#step:6:28

>node_modules/resize-observer-polyfill/src/index.d.ts(19,18): error TS2717: Subsequent property declarations must have the same type.  Property 'contentRect' must be of type 'DOMRectReadOnly', but here has type 'DOMRectReadOnly'.